src/styles/main/card.scss

```diff
@@ -0,0 +1,103 @@
+.card {
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  background: #fff;
+  transition: transform 300ms ease;
+
+  &__images {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    margin: 0 0 4px;
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    color: $text-color;
+    transition: color 300ms ease;
+
+    &:hover {
+      color: #34568b;
+    }
```

Comment:
Card title should change color when the card itself is hovered, not when the title is hovered [Referece](https://github.com/mate-academy/layout_catalog/tree/master#:~:text=change%20the%20card%20title%20text%20color%20to%20%2334568b%20when%20the%20card%20is%20hovered%20(.card%3Ahover%20.card__title))